frameworks/hdfs/tests/config.py

```diff
@@ -20,58 +22,48 @@
 DOCKER_IMAGE_NAME = "mesosphere/hdfs-client:2.6.4"
 KERBERIZED_CLIENT_IMAGE_NAME = "nvaziri/kerberized-hdfs-client:dev"
 KERBERIZED_CLIENT_MEM_SIZE = 512
```

Comment:
Should we have to override the memory here? It is already specified in the `json` file?

Reply:
Yeah, can also be removed.